frontend/src/metabase/admin/permissions/selectors/confirmations.ts

```diff
@@ -157,38 +158,69 @@ export function getWillRevokeNativeAccessWarningModal(
   }
 }
 
-export function getRawQueryWarningModal(
```

Comment:
This isn't part of your diff, but does line 77 need a little love?

```typescript
    return t`The "${defaultGroup.name}" group may have access to a different set of ${entityType} than this group, which may give this group additional access to some ${entityType}.`;
```

In certain cases this may read "...a different set of null than this group" or "...a different set of database than this group"

Reply:
![CleanShot 2024-08-13 at 08 51 45@2x](https://github.com/user-attachments/assets/e12e9b5c-343e-464b-b1f3-997539fe3dca)

dang, sure enough... i was able to reproduce it. i'll find a fix either in this PR or another 👍